### DISCUSSION TITLE
Found a block but the Daily Alert says i haven't

### DISCUSSION THREAD

**Discussion by Macie73** [guydavis/machinaris#1006] [Q&A]
hi team,

recently changed over from a native Chia client to Machinaris docker image (one Farmer/Full Node with 2 remote harvester).
After a few days I got my first block! Indicated by the little guy emoji jumping up and done in the XCH farmed box.
However my daily alerts, which i've configured to be email to me didn't include this fact. I found the block last night after the previous Daily update.
It just says:
Hello farmer! 👋 Here's what happened in the last 24 hours:

Received ☘️: 0 XCH
Proofs 🧾: 160 found!
- 0 partials submitted 📑
- 0 blocks found 📦        # --> i would have expect this line to say "1"
Search 🔍: 
- average: 1.05s over 1445 searches
- over 5s: 6 occasions (0.4%)
- over 15s: 0 occasions (0.0%)
Plots 🌱: 1666
Eligible plots 🥇: 6.55 average
Skipped SPs ⚠️: Unknown

above is from the Harvester, the Farmer message didn't include it either:
Hello farmer! 👋 Here's what happened in the last 24 hours:

Received ☘️: 0 XCH
Proofs 🧾: None
- 472 partials submitted 📑
- 0 blocks found 📦
Search 🔍: 
- average: 0.00s over 1464 searches
- over 5s: 0 occasions (0.0%)
- over 15s: 0 occasions (0.0%)
Plots 🌱: 0
Eligible plots 🥇: 0.00 average
Skipped SPs ✅️: None

any idea why the block is not included?

thanks

**Comment by guydavis** [comment]
Good question.  Chia has made changes in the past that limited  the default alerting of 3rd party tools like Chiadog. Also, Chiadog (used for Alerting in Machinaris) hasn't seen any recent updates unfortunately. 

You could try switching your `log_level` under Settings | Farming to debug from info.  Generally, I just  keep an eye on the Wallets chart to see when a block is found.  Honestly, I don't  check more than monthly anymore.

**Comment by Macie73** [comment]
thanks for the tip Guy!
Indeed the Wallet chart did show it correctly so it all ended up in the
right place - i just wished it could have been included in the email alert,
but no matter I can get that from the pool as well.
I know i'm late to the Chia game (started only a year ago) but wanted to
tell you what a nice piece of software you've created with Machinaris. Keep
up the good work!
cheers
